osgp/platform/osgp-adapter-domain-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/domain/da/application/services/AdHocManagementService.java

```diff
@@ -99,4 +99,10 @@ public void handleGetDeviceModelResponse(final GetDeviceModelResponseDto getDevi
                 .build();
         this.webServiceResponseMessageSender.send(responseMessage, messageType);
     }
+
+    public void handleGetDataRequest(final String payload, final String deviceIdentification) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "deviceIdentification". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1172)

Reply:
Logged it